### PR TITLE
Add services documentation to docs.toit.io.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: docs/language/sdk/services.mdx [hunk 1]
```diff
@@ -0,0 +1,540 @@
...
+<!-- HIDDEN CODE   open --timeout [--if_absent]: return null -->
+```
+service_/RandomGeneratorServiceClient? ::= (RandomGeneratorServiceClient).open
+    --timeout=(Duration --s=2)
+    --if_absent=: null
+```
+
+## Serialization
+As part of the communication between clients and providers, messages
+are serialized in an efficient way. Messagse must be either `null`,
```
Comment (kasperl) [inline docs/language/sdk/services.mdx:269]:
```suggestion
are serialized in an efficient way. Messages must be either `null`,
```

File: docs/language/sdk/services.mdx [hunk 2]
```diff
@@ -0,0 +1,540 @@
...
+integers, booleans, floats, strings, byte arrays, lists or maps
+(with string keys). Any type that is not one of these must be converted
```
Comment (kasperl) [inline docs/language/sdk/services.mdx:271]: Is the string keys really a requirement?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: docs/language/sdk/services.mdx (hunk 2 above)
Comment (floitsch) [reply docs/language/sdk/services.mdx:271]: no.
done.

**Pull request merged**